sopel/bot.py

```diff
@@ -376,6 +376,10 @@ def reply(self, text, dest, reply_to, notice=False):
         else:
             self.say(text, dest)
 
+    def kick(self, nick, channel, text=''):
+        """Send an IRC KICK to a user in a channel."""
```

Comment:
Going to propose a bit of a doozy here, along the lines of other docstrings in this class. Wrapped it roughly in GitHub's editor, but you can wrap to a proper length in your editor before committing. 😸

```suggestion
        """Send an IRC KICK command.

        Within the context of a triggered callable, ``channel`` will default to the
        channel in which the call was triggered. If triggered from a private message,
        ``channel`` is required (or the call to ``kick()`` will be ignored).
        """
```